adagrams/game.py

```diff
@@ -1,11 +1,108 @@
+import random
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
 def draw_letters():
-    pass
+    
+    hand_of_letters = []
+    letter_count =  {letter: 0 for letter in LETTER_POOL}
+    letter_list = list(LETTER_POOL.keys())
+    while len(hand_of_letters) < 10: 
+        
+        letter_index = random.randint(0, len(letter_list) - 1)
+        letter = letter_list[letter_index]
+        
+        if letter_count[letter] < LETTER_POOL[letter]:
+            hand_of_letters.append(letter)
+            letter_count[letter] += 1
+
+    return hand_of_letters
+
+
+
+
 
 def uses_available_letters(word, letter_bank):
-    pass
+    letter_bank_copy = letter_bank [:]
+    word = word.upper()
```

Comment:
Nice work on ensuring that all the letters where of the same casing. You could also use `casefold`, which is used for caseless matching. You can find more information on the method [here](https://docs.python.org/3/library/stdtypes.html#str.casefold).

Also be mindful of re-assigning the parameter of a function. It can be a side effect of your function if the caller doesn't expect for the data that they passed in to be altered.